### PR TITLE
Horizon europe view

### PR DESCRIPTION
**Pull request by Zack-83**

### CODE REVIEW

**Review by thrichter-FD — APPROVED** [review]

**Review by dario-pilori — COMMENTED** [review]
Installed without problems on our internal test instance.

**Review by torsten-rathmann — APPROVED** [review]

**Comment by torsten-rathmann** [comment]
I think it's a good solution. Three minor issues I have still found:

- At the beginning of the view output I have found this text (example): { "title": "DMP_ _ _v ", "category": " ", "subject": " ", "description": " Klimtest ", "keywords": ["Jochen", "Olaf"], "contentStatus": "135", "abstract": " Testen von RDMO mit fiktivem Klimadatenprojekt " }
- Under question 34, a lonely "+." appeares at the end of the line if non-personal costs aren't given.
- Under question 37, a capital "The" appears in the mid of the sentence: "All decision related to preservation fall under the responsibility of The project coordinator"

File: rdmorganiser/questions/horizon-europe.xml [hunk 1]
```diff
@@ -3833,12 +4354,17 @@ Auf diese beiden Fragen beziehen sich alle Detailfragen bis zum Ende des Abschni
...
+		<verbose_name lang="es"/>
+		<verbose_name_plural lang="es"/>
 		<help lang="fr">Fondamentalement, la collecte, le traitement, l'archivage et la publication de données personnelles ne sont admissibles que lorsque le consentement éclairé de la personne concernée a été obtenu. Ce n'est que dans de très rares cas que cela n'est pas nécessaire. Si besoin, veuillez contacter le délégué à la protection des données de votre institution pour obtenir des informations sur ces conditions nécessaires de consentement.</help>
-		<text lang="fr">Dans quelle mesure le «?consentement éclairé?» des personnes concernées est-il obtenu??</text>
+		<text lang="fr">Dans quelle mesure le «?consentement éclairé?» des personnes concernées est-il obtenu&#8239;?</text>
 		<default_text lang="fr"/>
 		<verbose_name lang="fr"/>
 		<verbose_name_plural lang="fr"/>
-		<help lang="it">.....</help>
+		<help lang="it">In linea di principio, la collezione, il trattamento, l'archiviazione e la pubblicazione di dati personali non sono consentiti, a meno che non sia stato ottenuto il &quot;consenso informato&quot; delle persone coinvolte.</help>
```
Comment (Zack-83) [inline rdmorganiser/questions/horizon-europe.xml:4367]:
```suggestion
		<help lang="it">In linea di principio, la collezione, il trattamento, l'archiviazione e la pubblicazione di dati personali non sono consentiti, a meno che non sia stato ottenuto il "consenso informato" delle persone coinvolte.</help>
```

File: rdmorganiser/questions/horizon-europe.xml [hunk 2]
```diff
@@ -3833,12 +4354,17 @@ Auf diese beiden Fragen beziehen sich alle Detailfragen bis zum Ende des Abschni
...
 		<verbose_name lang="de"/>
 		<verbose_name_plural lang="de"/>
+		<help lang="es">Básicamente, la recopilación, procesamiento, archivo y publicación de datos personales solo es admisible cuando se ha obtenido el "consentimiento informado" de la persona en cuestión.</help>
+		<text lang="es">¿En qué medida se obtiene el &quot;consentimiento informado&quot; de las personas interesadas?</text>
+		<default_text lang="es"/>
+		<verbose_name lang="es"/>
+		<verbose_name_plural lang="es"/>
 		<help lang="fr">Fondamentalement, la collecte, le traitement, l'archivage et la publication de données personnelles ne sont admissibles que lorsque le consentement éclairé de la personne concernée a été obtenu. Ce n'est que dans de très rares cas que cela n'est pas nécessaire. Si besoin, veuillez contacter le délégué à la protection des données de votre institution pour obtenir des informations sur ces conditions nécessaires de consentement.</help>
-		<text lang="fr">Dans quelle mesure le «?consentement éclairé?» des personnes concernées est-il obtenu??</text>
+		<text lang="fr">Dans quelle mesure le «?consentement éclairé?» des personnes concernées est-il obtenu&#8239;?</text>
```
Comment (Zack-83) [inline rdmorganiser/questions/horizon-europe.xml:4363]:
```suggestion
		<text lang="fr">Dans quelle mesure le «&#8239;consentement éclairé&#8239;» des personnes concernées est-il obtenu&#8239;?</text>
```

File: rdmorganiser/questions/horizon-europe.xml [hunk 3]
```diff
@@ -3877,6 +4403,11 @@ Auf diese beiden Fragen beziehen sich alle Detailfragen bis zum Ende des Abschni
 		<default_text lang="de"/>
 		<verbose_name lang="de"/>
 		<verbose_name_plural lang="de"/>
+		<help lang="es"/>
+		<text lang="es">Si no se obtiene el &quot;consentimiento informado&quot;, indique las razones para no hacerlo.</text>
+		<default_text lang="es"/>
+		<verbose_name lang="es"/>
+		<verbose_name_plural lang="es"/>
 		<help lang="fr"/>
 		<text lang="fr">Si aucun «?consentement éclairé?» n'est obtenu, veuillez indiquer les raisons du refus.</text>
```
Comment (Zack-83) [inline rdmorganiser/questions/horizon-europe.xml:4412]:
```suggestion
		<text lang="fr">Si aucun «&#8239;consentement éclairé&#8239;» n'est obtenu, veuillez indiquer les raisons du refus.</text>
```

**Pull request merged**